streams/mx.m3u

```diff
@@ -29,8 +29,8 @@ https://5e50264bd6766.streamlock.net/canal442/videocanal442/playlist.m3u8
 https://5e50264bd6766.streamlock.net/canal44/videocanal44/playlist.m3u8
 #EXTINF:-1 tvg-id="CanalCatorce.mx" status="online",Canal Catorce (720p)
 https://d3nljkrx6mjqra.cloudfront.net/out/v1/1b9d9efd27814b3b8dc570113ae54409/index.m3u8
-#EXTINF:-1 tvg-id="CanaldelasEstrellas.mx" status="error",Canal de Las Estrellas (480p)
-http://170.83.242.153:8000/play/a012
+#EXTINF:-1 tvg-id="CanaldelasEstrellas.mx",Canal de Las Estrellas (480p)
+http://edge03-fdo-py.cvattv.com.ar/live/c6eds/Canal_de_las_estrellas/SA_SAGEMCOM/Canal_de_las_estrellas.m3u8
```

Comment:
<img width="758" alt="image" src="https://user-images.githubusercontent.com/7253922/159785591-6dcd9cc0-46eb-4ffe-a9db-5d5d336afdfa.png">

Reply:
Works in my case, tested with VLC.
![vlcsnap-2022-03-23-22h05m20s206](https://user-images.githubusercontent.com/30985701/159795617-2c735b1d-6049-4e83-92fe-58a24aa533fa.png)